docs/admin/kubeadm.md

```diff
@@ -156,6 +156,16 @@ names `<service_name>.<namespace>.svc.cluster.local`. You can use the
 update the `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` file
 accordingly else DNS will not function correctly.
 
+**Note**: Although this flag does have an affect currently (it's needed for the
```

Comment:
suggest: This flag has effect (it's needed for...), but not as you might expect, since you will have to...